transcom/mymove
# B-21049

## B-21049
[Agility B-21049](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1007783)
[Integration PR](https://github.com/transcom/mymove/pull/14003)

## Summary


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a TOO
3. Search for and select a customer
4. Supporting Documents
5. Upload a PDF or image file
6. To simulate the AWS S3 delay, delete the newest file from mymove/tmp/storage/user/*/uploads/
7. Refresh the Supporting Documents page
8. see the File Not Found message

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
